app/views/questions/triage_income_types/edit.html.erb

```diff
@@ -9,13 +9,13 @@
 
   <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card" } do |f| %>
     <div class="form-card__stacked-checkboxes spacing-above-0">
-      <%= f.cfa_checkbox(:income_rental, t('.income_types.rental'), options: { checked_value: "yes", unchecked_value: "no" }) %>
-      <%= f.cfa_checkbox(:income_farm, t('.income_types.farm'), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:income_type_rent, t('.income_types.rental'), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:income_type_farm, t('.income_types.farm'), options: { checked_value: "yes", unchecked_value: "no" }) %>
       <%= f.cfa_checkbox(:none_of_the_above, t("general.none_of_the_above"), options: { id: "none__checkbox", checked_value: "yes", unchecked_value: "no" }) %>
     </div>
-  <% end %>
 
-  <div class="spacing-above-60">
-    <%= link_to t('general.continue'), next_path, class: "button button--primary button--wide text--centered" %>
-  </div>
+    <div class="spacing-above-60">
+      <%= f.submit t('general.continue'), class: "button button--primary button--wide text--centered" %>
```

Comment:
@tgrathwell dagnabit sorry we missed this one when you and I were working on this story - Asheesh